src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -783,8 +783,8 @@ private void CheckStake(Block block, ChainedBlock chainTip)
             }
 
             this.logger.LogTrace("Worker #{0} found the kernel.", workersResult.KernelFoundIndex);
-
-            long reward = fees + this.posConsensusValidator.GetProofOfStakeReward(chainTip.Height);
+	        //Get reward for newly created block
+	        long reward = fees + this.posConsensusValidator.GetProofOfStakeReward(chainTip.Height+1);
```

Comment:
– don't use Tabs, use spaces
– Use Ctrl+K+D to let visual studio format your code properly, which would mean spaces around the +